xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPRendererDMA.cpp

```diff
@@ -34,6 +35,9 @@ CRPBaseRenderer* CRendererFactoryDMA::CreateRenderer(const CRenderSettings& sett
 
 RenderBufferPoolVector CRendererFactoryDMA::CreateBufferPools(CRenderContext& context)
 {
+  if (!CBufferObjectFactory::HasRegisteredBufferObjects())
+    return {};
```

Comment:
> @garbear please check this commit and let me know if you think that is the correct place to check this.

It looks like returning empty here is the expected behavior: https://github.com/xbmc/xbmc/blob/4e57e432f23299ee78fcfa105f2d96989a7065e3/xbmc/cores/RetroPlayer/process/RPProcessInfo.cpp#L46